landoui/treestatus.py

```diff
@@ -65,130 +63,77 @@ def build_recent_changes_stack(
                     if change["trees"][0]["last_state"]["current_tags"]
                     else ReasonCategory.NO_CATEGORY.value
                 ),
-                who=change["who"],
-                when=change["when"],
             ),
             change,
         )
-        for change in response["result"]
+        for change in recent_changes_data
     ]
 
 
-@treestatus_blueprint.route("/treestatus/", methods=["GET"])
+@treestatus_blueprint.route("/treestatus/", methods=["GET", "POST"])
 def treestatus():
-    """Display the status of all the current trees."""
-    token = get_phabricator_api_token()
-    api = LandoAPI(
-        current_app.config["LANDO_API_URL"],
-        auth0_access_token=session.get("access_token"),
-        phabricator_api_token=token,
-    )
+    """Display the status of all the current trees.
+
+    This view is the main landing page for Treestatus. The view is a list of all trees
+    and their current statuses. The view of all trees is a form where each tree can be
+    selected, and "Update trees" passes the selection along to the tree updating form.
```

Comment:
What is "Update trees"? Maybe be more explicit here (e.g., "submitting the update tree form").

Reply:
"Update trees" is the text on the button used to display the form modal. I'll change this to `clicking "Update trees"`.